core/include/gnuradio-4.0/Block.hpp

```diff
@@ -399,6 +413,19 @@ struct Block : protected std::tuple<Arguments...> {
     using StrideControl              = ArgumentsTypeList::template find_or_default<is_stride, Stride<0UZ, true>>;
     constexpr static bool blockingIO = std::disjunction_v<std::is_same<BlockingIO<true>, Arguments>...> || std::disjunction_v<std::is_same<BlockingIO<false>, Arguments>...>;
 
+    template<typename T>
+    auto &
+    getArgument() {
```

Comment:
Do we need this? seems to to be an alias for the tupple get...

Reply:
The fact that the Block is a tuple is an internal thing (we should at some point see what should be public/protected/private in GR), this is API to access the arguments from outside of the block.

Reply:
Could this be linked to the change of `struct Block` -> `class Block` in this PR?

Reply:
No, the inheritance from tuple was protected before this change. The struct -> class change just influences the default access if none specified.

In the case of block, it has public...private, so it should be a class and private by default.

For tuple-inheritance, it should even (IMO) be private as it is an implementation detail and not something that people should rely on.